public/index.html

```diff
@@ -24,12 +22,16 @@
       work correctly both with client-side routing and a non-root public URL.
       Learn how to configure a non-root public URL by running `npm run build`.
     -->
-    <title>React App</title>
-  </head>
-  <body>
-    <noscript>You need to enable JavaScript to run this app.</noscript>
-    <div id="root"></div>
-    <!--
+  <link rel="preconnect" href="https://fonts.googleapis.com">
+  <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
```

Comment:
I have always wondered if these lines are truly necessary :thinking:

Reply:
These lines help with page load performance because that way you notify the origin (`fonts.gapis` in this case) that these assets are being loaded, so the resources are loaded a bit early in the page.
Here's a good article explaining that more in detail: https://css-tricks.com/using-relpreconnect-to-establish-network-connections-early-and-increase-performance/

https://css-tricks.com/using-relpreconnect-to-establish-network-connections-early-and-increase-performance/